docs/references/vue/overview.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Clerk Vue SDK
+description: Learn how to use Clerk to quickly and easily add secure authentication and user management to your Vue application.
+---
+
+The Clerk Vue SDK is a wrapper around the Clerk JavaScript SDK. It is the recommended way to integrate Clerk into your Vue application.
+
+> [!IMPORTANT]
+> If you're using Nuxt, we recommend installing the `@clerk/nuxt` package since it's specifically built as a Nuxt module and includes backend integration. See the [Nuxt](/docs/references/nuxt/overview) documentation for more information.
```

Comment:
This contains a dead link that will redirect to [this PR](https://github.com/clerk/clerk-docs/pull/1728)